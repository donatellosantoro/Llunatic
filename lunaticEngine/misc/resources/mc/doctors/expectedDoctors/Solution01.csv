#doctor
npi,name,hospital,email
5555,G. House,PPTH,house@ppth.com
_L212,_L218,Sacred Hearth,chase@sh.com

#prescription
rxid,patient,npiref
P1,J. Doe,_L212
P2,W. Smith,5555
P3,M. White,5555

