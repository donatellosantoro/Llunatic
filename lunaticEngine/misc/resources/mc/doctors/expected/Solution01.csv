#prescription
rxid,patient,npiref
P3,M. White,5555
P1,J. Doe,_L179
P2,W. Smith,5555

#doctor
npi,name,hospital,email
_L179,_L183,Sacred Hearth,chase@sh.com
5555,G. House,PPTH,house@ppth.com
